maintenance/pin.sh

```diff
@@ -6,4 +6,4 @@
 set -e
 set -x
 
-docker run --rm -t -v $PWD:/src -w /src python:3.11 maintenance/pin-helper.sh $@
+docker run --platform=linux/x86_64 --rm -t -v $PWD:/src -w /src python:3.11 maintenance/pin-helper.sh $@
```

Comment:
This is necessary for M1 (or other non-x86_64 machines), I guess?

Reply:
That's right. Just to make sure we're always compiling with the same platform.